DeepSkyStacker/StackRecap.cpp

```diff
@@ -209,7 +209,9 @@ void StackRecap::fillWithAllTasks()
 		bool				bSaveIntermediates;
 
 		ulNeededSpace = pStackingTasks->ComputeNecessaryDiskSpace();
-		ulFreeSpace = pStackingTasks->AvailableDiskSpace(CString((wchar_t*)strDrive.utf16()));
+		CString				strDriveCString;
+		strDriveCString = CString((wchar_t*)strDrive.utf16());
+		ulFreeSpace = pStackingTasks->AvailableDiskSpace(strDriveCString);
```

Comment:
The compiler errors if we pass a non-const reference to a temporary as an argument to a function (here the function `pStackingTasks->AvailableDiskSpace(...)`, therefore I had to explicitly create an intermediate variable.